package.json

```diff
@@ -36,7 +36,8 @@
     "electrode-redux-router-engine": "^1.0.0",
     "electrode-server": "^1.0.0",
     "electrode-static-paths": "^1.0.0",
-    "lodash": "^4.10.1"
+    "lodash": "^4.10.1",
+    "react-notify-toast": "^0.1.3"
```

Comment:
@ananavati i have moved to `dep` instead of `dev`: https://github.com/electrode-io/electrode-archetype-react-app/pull/62